partiql-spi/build.gradle.kts

```diff
@@ -29,19 +31,17 @@ tasks.shadowJar {
     configurations = listOf(project.configurations.shadow.get())
 }
 
+tasks.withType(DokkaTask::class.java) {
```

Comment:
It was, but now thinking about it. I'm going to revert this in favor of Dokka's Javadoc capabilities, since I believe the publishing plugin relies on it. That being said, once all public APIs move to Java, we should consider using Javadoc. As an anectode, I've found Dokka to be extremely slow, and if we don't use Kotlin as the public API, it'd be a good idea to move to Javadoc.

Reply:
Ack. A lot of this package's APIs are still in Kotlin so we probably need to retain Dokka for publishing. May be worth experimenting though if we can use some other way to publish java/kotlin docs.